### ISSUE TITLE
Language Feast: C#

### ISSUE THREAD

**Issue by Oleksii-Kshenskyi** [Oleksii-Kshenskyi/btagsil#26]
This specific issue is a little different from all others of this type. In its essence, this issue is similar to all others of the [Language Feast](https://github.com/Oleksii-Kshenskyi/btagsil/milestone/3) milestone: I'm going to be writing the same exact prototype as all other issues in it.

However, one difference is that the purpose of this prototype is different: I'm going to be writing it with a non-crossplatform technology: C# on Visual Studio, specifically. Also, I'm not going to be considering it as a possible language of implementation for BTAGSIL. The purpose of this specific prototype is for it to serve as an example of how to write this type of application in C# to my mentee. The point is for us both to write the same application differently, and then compare them and talk about them when they're done.
So TL;DR: it's an educational prototype instead of a research one. Even though the functionality is going to be exactly the same as all others, the purpose is different.

**Comment by Oleksii-Kshenskyi** [comment]
I've finished the prototype and showed/explained it to my mentee. With this, the issue can be closed.
